helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -224,7 +224,7 @@ public void testGetClusterTopologyAndFaultZoneMap() throws IOException {
   @Test(dependsOnMethods = "testGetClusterTopologyAndFaultZoneMap")
   public void testAddConfigFields() throws IOException {
     System.out.println("Start test :" + TestHelper.getTestMethodName());
-    String cluster = _clusters.iterator().next();
+    String cluster = "TestCluster_1";
```

Comment:
This fix could work, but it is not very intuitive for writing future tests. People may easily overlook the guideline. Do you think it's possible to reset the cluster back to normal after waged related tests? If it's not feasible, we may want to add more description for this guideline or not to name it simply as 0, 1, 2.

Reply:
TFTR. I think adding more description for these clusters might be a better option here. Let me update.